### ISSUE TITLE
Kuriyama et al. 2015

### ISSUE THREAD

**Issue by aaronmberger** [cgrandin/hake-assessment#53]
Andy,

Would you please add the below reference - I fixed citation in weight-at-age section.

Kuriyama, P.T. and 11 coauthors. 2015. An empirical weight-at-age approach reduces estimation bias compared to modeling parametric growth in integrated, statistical stock assessment models when growth is time varying.  Fish Res: doi:10.1016/j.fishres.2015.09.007.

**Comment by allan-hicks** [comment]
Here is the bibtex entry:

@article{Kuriyama2015,

title = "An empirical weight-at-age approach reduces estimation bias
compared to modeling parametric growth in integrated, statistical stock
assessment models when growth is time varying ",

journal = "Fisheries Research ",

volume = "",

number = "",

pages = " - ",

year = "2015",

note = "",

issn = "0165-7836",

doi = "http://dx.doi.org/10.1016/j.fishres.2015.09.007",

url = "http://www.sciencedirect.com/science/article/pii/S0165783615300837",

author = "Peter T. Kuriyama and Kotaro Ono and Felipe Hurtado-Ferro and
Allan C. Hicks and Ian G. Taylor and Roberto R. Licandeo and Kelli F.
Johnson and Sean C. Anderson and Cole C. Monnahan and Merrill B. Rudd and
Christine C. Stawitz and Juan L. Valero",

keywords = "Empirical weight-at-age information",

keywords = "Fisheries stock assessment",

keywords = "Time-varying growth",

keywords = "ss3sim",

keywords = "Stock Synthesis ",

abstract = "Abstract Somatic growth in fishes often varies through time.
Despite this, most stock assessments either fix or estimate a
time-invariant growth relationship because estimating time-varying growth
parameters can be data intensive and subject to multiple sources of bias.
Additionally, estimates of growth are often confounded with estimates of
selectivity, particularly if selectivity is also time varying.
Incorporating empirical weight-at-age (EWAA) information into assessments
is a little-studied alternative that accounts for time-varying growth, but
foregoes fixing or estimating growth and length–weight relationships.
However, this method requires annual measures of fish weights at each age,
and missing values must therefore be interpolated. We used Stock Synthesis
in a simulation-testing framework to compare the effect of estimating a
single time-invariant growth curve, time-varying growth curves, and
incorporating \{EWAA\} information on management quantities and parameter
estimates. We ran simulations across two fish life histories (hake-like and
rockfish-like) and three data cases (data-rich, data-rich with a
late-starting survey, and data-moderate). We found that when growth was
time invariant, the \{EWAA\} approach was unbiased but had twice the median
average relative error compared to a model that estimated growth from age
and length data. However, for data-rich cases with time-varying growth, the
\{EWAA\} method resulted in more accurate estimates of spawning stock
biomass compared to the approach that estimated time-invariant and
time-varying growth parameters, as evidenced by at least a five-fold
reduction in range of median relative errors. The magnitude of this effect
was greatest for the long-lived, slow-growing life history. For the
relatively fast-growing species, estimates from the \{EWAA\} method were
particularly sensitive to interpolating missing values. "

}
